### PR TITLE
Update outdated event id doc

### PR DESCRIPTION
**Pull request by halfprice**
## Description 

As title says

## Test plan 

How did you test the new or updated feature?

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

File: crates/sui-open-rpc/spec/openrpc.json [hunk 1]
```diff
@@ -6352,7 +6352,7 @@
         ]
       },
       "EventID": {
-        "description": "Unique ID of a Sui Event, the ID is a combination of tx seq number and event seq number, the ID is local to this particular fullnode and will be different from other fullnode.",
+        "description": "Unique ID of a Sui Event, the ID is a combination of tx seq number and event seq number.",
```
Comment (bmwill) [inline crates/sui-open-rpc/spec/openrpc.json:6355]: change tx seq number to "transaction digest"

File: crates/sui-types/src/event.rs [hunk 2]
```diff
@@ -38,8 +38,7 @@ pub struct EventEnvelope {
     /// Move event's json value
     pub parsed_json: Value,
 }
-/// Unique ID of a Sui Event, the ID is a combination of tx seq number and event seq number,
-/// the ID is local to this particular fullnode and will be different from other fullnode.
+/// Unique ID of a Sui Event, the ID is a combination of tx seq number and event seq number.
```
Comment (bmwill) [inline crates/sui-types/src/event.rs:41]: here too

File: sdk/typescript/src/client/types/generated.ts [hunk 3]
```diff
@@ -290,8 +290,7 @@ export type SuiEventFilter =
 			Or: [SuiEventFilter, SuiEventFilter];
 	  };
 /**
- * Unique ID of a Sui Event, the ID is a combination of tx seq number and event seq number, the ID is
- * local to this particular fullnode and will be different from other fullnode.
+ * Unique ID of a Sui Event, the ID is a combination of tx seq number and event seq number.
```
Comment (bmwill) [inline sdk/typescript/src/client/types/generated.ts:293]: here too

**Review by bmwill — APPROVED** [review]

File: crates/sui-open-rpc/spec/openrpc.json (hunk 1 above)
Comment (halfprice) [reply crates/sui-open-rpc/spec/openrpc.json]: Done

**Pull request merged**